doc/source/guides/getting-started.rst

```diff
@@ -0,0 +1,97 @@
+.. _guide-getting-started:
+
+===============
+Getting started
+===============
+
+
+First of all: Add the Library to your Build
+===========================================
+
+The gini sdk is provided via `Cocoapods <http://www.cocoapods.org>`_.
+To install the gini sdk simply add the following repository to your Cocoapods installation
+
+.. code-block:: none
+
+    $ pod repo add gini-podspecs https://github.com/gini/gini-podspecs.git
+
+and include the pod in your Podfile
+
+.. code-block:: none
+
+    pod 'Gini-iOS-SDK'
+
+Then run
+
+.. code-block:: none
+
+    $ pod install
+    
+in your project directory and open the generated Xcode workspace.
+
+
+Integrating the Gini SDK
+========================
+
+
+The Gini SDK provides the ``GiniSDK`` class which is a façade to all functionality of the Gini SDK. We recommend using an
+instance of this class singleton-like. By saying singleton-like we mean that you somehow manage to create and keep
+one instance at application start. Instead of creating a new instance every time when you need to interact with the
+Gini API, you reuse this instance. This has the benefits that the SDK can reuse sessions between requests to the
```

Comment:
This has the benefit that the SDK can reuse sessions between requests to the Gini API, which may save a significant number of HTTP requests.